.buildkite/pipeline.yml

```diff
@@ -1,5 +0,0 @@
-steps:
```

Comment:
I think we'll also need to disable buildkite *somewhere* because it currently shows as a failing check in github.

Reply:
Done, I've removed it from the branch protection and archived the pipeline.